app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%= @mechanic.name %>'s Show Page</h1>
+
+<h4><%= @mechanic.name %>'s Details:</h4>
+<p>Name: <%= @mechanic.name %></p>
+<p>Years of Experience: <%= @mechanic.years_experience %></p></br>
+
+<h4>Rides Currently Servicing:</h4>
+<div class="ride-list">
+  <ul>
+    <% @mechanic.working_ride_list.each do |ride_name| %>
```

Comment:
Here is where I was saying that I don't think that you need to necessarily create a method in your model (although you can) because you just iterate over `@mechanic.rides.each do |ride|` and then call `ride.name` in the block. Not that what you did is wrong but might not be necessary.